kicker/app/workflows/transfer-arkivuttrekk-to-bevaring.yaml

```
@@ -178,3 +221,52 @@ spec:
             value: "{{inputs.parameters.iam_access_key_id}}"
           - name: DATASETT_ID
             value: "{{inputs.parameters.datasett_id}}"
+
+    - name: exit-handler
+      dag:
+        tasks:
+          - name: task-save-dataset-in-mottak
+            template: save-dataset-in-mottak
+            arguments:
+              artifacts:
+                - name: optional_env_file
+                  from: "{{workflow.outputs.artifacts.env_file}}"
+                  optional: true
+
+    - name: save-dataset-in-mottak
+      retryStrategy:
+        limit: 4
+        backoff:
+          duration: "60"
+      inputs:
+        artifacts:
+          - name: optional_env_file
+            path: /optional_env_file
+            optional: true
+      volumes:
+        - name: tmp
+          emptyDir: { }
+      container:
+        image: arkivverket.azurecr.io/da-mottak/save-dataset-in-mottak:{{workflow.parameters.SAVE_DATASET_IN_MOTTAK_TAG}}
+        imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: 500m
+            memory: 200Mi
+          limits:
+            memory: 300Mi
```

Denne trenger ikke så mye ressurser ser jeg for meg

```suggestion
        resources:
          requests:
            cpu: 100m
            memory: 64Mi
          limits:
            cpu: 300m
            memory: 192Mi
```